local_development.yml

```diff
@@ -157,6 +157,8 @@
           collection: cluster
         - repo: git@github.com:cloudera-labs/cloudera.exe.git
           collection: exe
+        - repo: git@github.com:ansible-collections/cloud.terraform.git
```

Comment:
We shouldn't need to clone this repo. An extra `ansible_galaxy` task to install it would work.

Reply:
@wmudge Something like this?

- name: Install the cloud.terraform collection dependencies
  community.general.ansible_galaxy_install:
    type: collection
    name: cloud.terraform
    source: git+https://github.com/ansible-collections/cloud.terraform.git
    state: present
    force: true
  tags: collections

Seems that you are not using a requirements.txt file so I could add this task to the local_deployment role

Reply:
For now -- that is, on the `main` branch -- we can leave the dependency as inline. The upcoming 2.x release on `devel` has (or will have) this dependency in the base image `requirements.yml` file for `ansible-builder`. The local development build process crawl these requirements and install as needed.

Looking at [`community.general.ansible_galaxy_install`](https://docs.ansible.com/ansible/latest/collections/community/general/ansible_galaxy_install_module.html#parameters), I don't believe you will need `source`, rather you can append the version number (`1.1.1`):

```yaml
community.general.ansible_galaxy_install:
    type: collection
    name: cloud.terraform:1.1.1
```